plugins/utils/hop.nix

```diff
@@ -0,0 +1,226 @@
+{
+  lib,
+  helpers,
+  config,
+  pkgs,
+  ...
+}:
+with lib;
+  helpers.neovim-plugin.mkNeovimPlugin config {
+    name = "hop";
+    originalName = "hop.nvim";
+    defaultPackage = pkgs.vimPlugins.hop-nvim;
+
```

Comment:
We should add a plugin description telling that it does not defined any keybinds, and link to the readme on how to add them. We could also add helpers to do that outside of settings, but this could be complex due to the large number of options there seems to have. Not sure what is the best way.